src/DaPigGuy/PiggyCustomEnchants/PiggyCustomEnchants.php

```diff
@@ -70,16 +70,17 @@ public function onDisable(): void
         foreach ($this->getServer()->getOnlinePlayers() as $player) {
             foreach ($player->getInventory()->getContents() as $slot => $content) {
                 foreach ($content->getEnchantments() as $enchantmentInstance) {
+                    /** @var ToggleableEnchantment $enchantment */
                     $enchantment = $enchantmentInstance->getType();
-                    if ($enchantment instanceof ToggleableEnchantment) {
+                    if ($enchantment instanceof CustomEnchant && $enchantment->canToggle()) {
```

Comment:
Why add the typehint for the enchant if you then check if it's something else?

Reply:
Getting PHPStorm to shut up